Update help.rs: bash completions instructions

rickhg12hs opened the pull request:
`>>` --> `>`

Closes https://github.com/rust-lang/rustup/issues/4375.

rami3l commented:
（Please don't mind the CI errors on Windows; our pipeline is experiencing some issues since ~1h ago)

rami3l reviewed (approved):
Thanks for the patch, and LGTM!

PS: You would need to fix the snapshot error as shown in the Linux CI log; the aforementioned CI outage only concerns Windows.

rami3l commented:
> You would need to fix the snapshot error as shown in the Linux CI log; the aforementioned CI outage only concerns Windows.

@rickhg12hs Also, please squash all the relevant changes in a single commit :)

rami3l commented:
@rickhg12hs I'm really sorry for the back and forth, but could you please rebase this PR and squash it to a single commit? We're using a merge queue so the merge commit is not required. Thanks!

rickhg12hs commented:
@rami3l Re: rebase/squash

Sure, as soon as I learn what that is and how to do it.  😳

rickhg12hs committed:
Update help.rs: bash completions instructions
`>>` --> `>`

Update rustup_completions_cmd_help_flag_stdout.toml
Fix bash completions test.

rickhg12hs committed:
Merge branch 'rust-lang:master' into bash-completions

rickhg12hs commented:
@rami3l Apparently I have a lot to learn.  Should I start over?

rami3l commented:
@rickhg12hs I don't think you need to start over; just do another rebase onto master and it should be fine, as the merge commit will be flushed.

You may need to make the rebase [interactive](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d) and keep the single commit you want. That is also how you would do a squash normally.

rickhg12hs commented:
I wasn't planning or expecting on closing this, but here I am (perplexed).  Resubmitting as https://github.com/rust-lang/rustup/pull/4378
